kernel/kernel-impl/pom.xml

```diff
@@ -397,6 +397,10 @@
             <artifactId>json-simple</artifactId>
             <version>${json.simple.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.sakaiproject</groupId>
+            <artifactId>sakai-citations-api</artifactId>
+        </dependency>
```

Comment:
It is generally not a good idea for kernel to depend on other components, would be ideal to rethink this

Reply:
Yes, that makes sense. 
I can think about two possibilities:
1.   implementing Observer in citationService and have the contentService post an event.
2.   contentService.Storage uses hardCoded sql (ContentServiceSql) Statements to clean the citation tables

Is any of these  suggestions  valid and feasible?